lgc/patch/PatchResourceCollect.cpp

```diff
@@ -2583,13 +2594,18 @@ void PatchResourceCollect::reassembleOutputExportCalls() {
       }
       continue;
     } else {
-      // To be re-assembaled call
+      // To be re-assembled call
       m_deadCalls.push_back(call);
     }
 
     const unsigned newLoc = mapIter->second.getLocation();
     auto &elementsInfo = elementsInfoArray[newLoc];
-    unsigned elemIdx = mapIter->second.getComponent() * 2 + mapIter->second.isHighHalf();
+
+    const unsigned origComponentIdx = mapIter->second.getComponent();
+    // Update the first components used
+    elementsInfo.baseOrigComponentIdx = std::min(elementsInfo.baseOrigComponentIdx, origComponentIdx);
+
```

Comment:
`mapIter->first` represents the original `InOutLocationInfo` and `mapIter->second` represents the mapped `InOutLocationInfo`. I think it's easy to get confused by using the `orig` prefix on the mapped `InOutLocationInfo`.

Reply:
I renamed it to `baseMappedComponentIdx`. Hope it matches the naming of the other code better now.